app/src/main/java/com/arctouch/codechallenge/util/MovieImageUrlBuilder.kt

```diff
@@ -1,17 +1,32 @@
 package com.arctouch.codechallenge.util
 
+import android.widget.ImageView
+import com.arctouch.codechallenge.R
 import com.arctouch.codechallenge.api.TmdbApi
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.drawable.DrawableTransitionOptions
+import com.bumptech.glide.request.RequestOptions
 
 private val POSTER_URL = "https://image.tmdb.org/t/p/w154"
 private val BACKDROP_URL = "https://image.tmdb.org/t/p/w780"
 
+//Glide controller
 class MovieImageUrlBuilder {
+    companion object {
```

Comment:
Why do these functions need to be in a companion object?

Reply:
I use companion object in this case to prevent create object every time i need to use this function. But I can do as the NetworkUtils, as a singleton.